manage_breast_screening/participants/tests/test_presenters.py

```diff
@@ -59,13 +59,28 @@ def test_presented_values(self, participant):
         assert result.url == f"/participants/{participant.pk}/"
 
     @pytest.mark.parametrize(
-        "background_id", Ethnicity.non_specific_ethnic_backgrounds()
+        "background_id,expected_display",
+        [
+            ("any_other_white_background", "any other White background"),
+            (
+                "any_other_mixed_or_multiple_ethnic_background",
+                "any other mixed or multiple ethnic background",
+            ),
+            ("any_other_asian_background", "any other Asian background"),
+            (
+                "any_other_black_african_or_caribbean_background",
+                "any other Black, African or Caribbean background",
+            ),
+            ("any_other_ethnic_background", "any other ethnic group"),
+        ],
     )
-    def test_any_other_ethnic_background(self, participant, background_id):
+    def test_any_other_ethnic_background(
```

Comment:
[minor] not overly bothered about this but this test could be broadened to test the non special values as well, and None